ext/smoke/repos/run-smoke-test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+USAGE="USAGE: $0 <master-vm1> <master-vm2> <agent-vm1> <agent-vm2> <agent-version> <server-version> <puppetdb-version>"
+domain=".delivery.puppetlabs.net"
+
+master_vm1="$1"
+master_vm2="$2"
+agent_vm1="$3"
+agent_vm2="$4"
+agent_version="$5"
+server_version="$6"
+puppetdb_version="$7"
+
+if [[ -z "${master_vm1}" || -z "${master_vm2}" || -z "${agent_vm1}" || \
+      -z "${agent_vm2}" || -z "${agent_version}" || -z "${server_version}" || \
+      -z "${puppetdb_version}" ]]; then
+  echo "${USAGE}"
+  exit 1
+fi
+
+# Append domains after validation.
+master_vm1="$1$domain"
+master_vm2="$2$domain"
+agent_vm1="$3$domain"
+agent_vm2="$4$domain"
+
+echo "##### Setting up masters..."
+./steps/setup-masters.sh ${master_vm1} ${master_vm2} ${agent_version} ${server_version} ${puppetdb_version}
+
+# One agent starts with master 1, one agent starts with master 2
+echo "##### Master 1 (PuppetDB Module) + Agent 1"
+./steps/setup-agent.sh          ${master_vm1} ${agent_vm1} ${agent_version}
+./steps/run-validation-tests.sh ${master_vm1} ${agent_vm1}
+echo "##### Master 2 (PuppetDB Package)+ Agent 1"
```

Comment:
If you insist on testing the combinations, you should bring it up in the release retro. Until everybody agrees this is the way we're smoke testing things, let's stick to Master 1 - Agent 1, Master 2 - Agent 2. (The agents are separate so that each master would have their own agent, start from a clean slate.). If we stick to that, the labels should just be changed to "Master (PuppetDB Module) + Agent".